config/config.example.yml

```diff
@@ -239,6 +239,13 @@ https_only: false
 ##
 #statistics_enabled: false
 
+##
+## Maximum videos to be loaded into the playlist before the video playing
+##
+## Accepted values: a positive integer
+## Default: 100
+##
+max_lookback: 100
```

Comment:
Imo, another config option is not necessary. The server should always load 100 videos, and, if required, we load more of them with XHR when the playlist widget is scrolled.

Reply:
I've grown to like the 200 videos to be honest... But don't really have a strong opinion on this matter. I did it out of a suggestion from before. Maybe we can tackle this in another PR/Issue?

Reply:
Yeah, that was @syeopite, IIRC. @unixfox @TheFrenchGhosty what are your thoughts on this?

Reply:
Yeah it was originally at 20 which seems absurdly low to me. 

> The server should always load 100 videos, and, if required, we load more of them with XHR when the playlist widget is scrolled.

This is a great idea! The current behavior of looking back 100 videos is what's implemented on YouTube itself, but maybe we shouldn't restrict users of Invidious to the same constraint. 

However imho, I think loading 100 videos in both directions is enough for most use cases. If you need to get to an earlier video, then it's really just faster to go onto the actual playlist page itself.

Reply:
Yeah. Loading more than 100 videos will only be useful when we'll rework the playlist widget to reduce the size of the thumbnails and list indices, too.